Added Chantelle's meshes + Prefabs + new test scene

Schwaembaem opened the pull request:
Added low poly mesh variants of Chantelle's meshes, made new prefabs, deleted faulty meshes (Aloe_low)

Schwaembaem committed:
Added Chantelle's meshes + Prefabs + new test scene

Added low poly mesh variants of Chantelle's meshes, made new prefabs, deleted faulty meshes (Aloe_low)

Schwaembaem committed:
Added low poly Meshes

+ prefab prep for GD

mysticalfoxie commented:
Eine FBX File ist leider corrupt. Kann ich so leider nicht approven. :/ 

### Probleme mit der Geometrie
* `SM_Vending Machine.fbx > Side` - 6 Polygone
* `SM_Cabinet_Drawers.fbx > Cabinet Drawer_RoundGlass.002` - 2 Polygone
* `SM_Cabinet_Drawers.fbx > Cabinet Drawer_RoundGlass` - 2 Polygone
> Insgesamt 6 Polygone sind mit sich selbst gekreuzt (self-intersecting) und wurden entfernt.

### Probleme mit Normal Maps
* `Pot` hat keine Normal Map.
* `Chopstick_Holder` hat keine Normal Map.
* `Pipe_middle.002` hat keine Normal Map.
* `Pipe_middle.001` hat keine Normal Map.

![image](https://github.com/user-attachments/assets/2b96b8cb-ef92-449d-a86c-117af5bb85f4)

### Problem mit einem Prefab Asset
* `PRE__BubbleTea_Machine (2) Variant` -> Referenz zerstört.
* Außerdem bitte Namings anpassen. `PRE_Bubble Tea Machine 01`, `PRE_Bubble Tea Machine 02`, ...
* 
![image](https://github.com/user-attachments/assets/2178df30-e6ee-4e28-95f1-fe26eeb00be1)

Schwaembaem committed:
Added fixed meshes to fix the console errors in Unity

All Meshes updated that have given us problems; vending machine deleted; no fast fix

Schwaembaem committed:
deleted vending machine prefab

mysticalfoxie committed:
Merge branch 'refs/heads/develop' into art/sabi/test-meshes

mysticalfoxie committed:
Merge branch 'refs/heads/develop' into art/sabi/test-meshes

mysticalfoxie committed:
Merge branch 'develop' into art/sabi/test-meshes

mysticalfoxie committed:
Merge branch 'develop' into art/sabi/test-meshes

mysticalfoxie commented:
Wird jetzt trotzdem gemerged, weil dieser Branch in einem anderen dringend benötigt wird. 
> -> Issue erstellt im Issue Tab, damit wir das Problem nicht unaddressiert lassen.
